### ISSUE TITLE
2024-07-14 snapshot stats and major changes

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-releases#91]
stats are here: https://docs.google.com/spreadsheets/d/1kTSYUTrgDXpXlXJpekG3ifRCbjjIqinnI3vH3DgU_BU/edit?gid=0#gid=0

**Comment by pgaudet** [comment]
Major changes: 

**BY ORGANISM**
**Human:**  + 82870
* +18760 IDA
* +60678 IPI  >> many new IntAct& ComplexPortal
*  -9848  gene products
* Are these to isoforms? 
  * 1466 "-PRO" in col 17
  * 45,000 have '-' (ie, isoforms or UniProt "chains") 
  * New RNAs that are now not being converted to 'gene product': 
     * pre_miRNA: +3257
     * lncRNA: +22082
     * rRNA: -6965


**AmiGO versus AmiGO-staging:** 
* 707156 human annotations in AmiGO
* 769603 human annotations in AmiGO-staging
* diff is 62,447;  not 82,870 as shown in the stats


**Source files**
* 2024-06-17 release: 
  * source: 705016 lines (note that this includes headers but they are the same) 
  * GOC-produced file: 707209 lines
 * 2024-07-14 snapshot: 
  * source: 767735 lines 
  * GOC-produced file: 769659  lines

So, I am not sure why the stats are inconsistent. But the data in the files looks right relative to the upstream.
In conclusion, Human data looks OK


----
**Mouse** : + 67644
* +4768 IDA	 >> AmiGO versus staging: UniProt + ~ 1000 new mouse IDAs. But there is not 4700 new IDAs
* +3595  IPI  >> many new IntAct& ComplexPortal
* +37703 ISO >> pipeline moved to GOC
* -13447 IEA

**Mouse IEAs: 
AmiGO:** 69359
- EC	1115
- InterPro	17944
- UniProtKB-KW	50300
- (all assigned_by MGI)

**Staging**: 55912 
- Ensembl	26172
- UniProt	20516
- InterPro	5180
- GOC	2851
- RHEA	1193

Much better! 	
----
**Drosophila:** + 4427
* 1542 IDA
* 2500 IMP
* 145 IGI
----
**BY GROUP** 
**MGI** : -138973 / 179719 | -77.33%
** **HPA** : + 14103 / 24306 | 58.02%

**Comment by pgaudet** [comment]
**CHANGES IN REFERENCES AND PMIDS BY GROUP**
GOC | -843 / 1000 | -84.30% | -419 / 499 | -83.97%
-- | -- | -- | -- | --
YuBioLab | -131 / 259 | -50.58% | 0 / 131 | 0.00%
NTNU_SB | -251 / 564 | -44.50% | -12 / 531 | -2.26%
DFLAT | -18 / 47 | -38.30% | 0 / 45 | 0.00%
ParkinsonsUK-UCL | -269 / 829 | -32.45% | 2 / 704 | 0.28%
SynGO-UCL | -9 / 33 | -27.27% | -1 / 21 | -4.76%
Alzheimers_University_of_Toronto | -15 / 57 | -26.32% | 1 / 48 | 2.08%
GOC-OWL | -14 / 56 | -25.00% | -7 / 28 | -25.00%
BHF-UCL | -1385 / 5616 | -24.66% | 16 / 5193 | 0.31%


----
**CHANGES IN REFERENCES AND PMIDS BY TAXON**
taxon | references | % references | pmids | % pmids
-- | -- | -- | -- | --
NCBITaxon:9606\|Homo sapiens | 1277 / 59664 | 2.14% | 1115 / 46960 | 2.37%
NCBITaxon:10116\|Rattus norvegicus | 38 / 49838 | 0.08% | 18 / 24695 | 0.07%
NCBITaxon:10090\|Mus musculus | -14473 / 36058 | -40.14% | 241 / 34595 | 0.70%


**This is because internal MGI reference IDs have been removed.** >> OK.

**Comment by pgaudet** [comment]
**CHANGES IN ANNOTATED BIOENTITIES BY FILTERED TAXON AND BY BIOENTITY TYPE (ALL)**   
taxon | protein | % protein | gene_product | % gene_product
-- | -- | -- | -- | --
NCBITaxon:9031\|Gallus gallus | 14 / 25514 | 0.05% | -542 / 2 | -27100.00%
NCBITaxon:9615\|Canis lupus familiaris | 324 / 33820 | 0.96% | -764 / 3 | -25466.67%
NCBITaxon:9823\|Sus scrofa | 53 / 26619 | 0.20% | -1640 / 22 | -7454.55%
NCBITaxon:9913\|Bos taurus | 6011 / 28331 | 21.22% | -808 / 25 | -3232.00%
NCBITaxon:4896\|Schizosaccharomyces pombe | 1 / 5040 | 0.02% | -16 / 1 | -1600.00%
NCBITaxon:9606\|Homo sapiens | 31 / 19752 | 0.16% | -9848 / 706 | -1394.90%


**This is because we fixed GORULE-0000001 and we now recognize more entities** (for example lncRNAs are not converted to 'gene product' anymore). (see https://github.com/geneontology/go-site/issues/2246) >>> OK.

**Comment by pgaudet** [comment]
**CHANGES IN ANNOTATIONS BY TAXON** 
Taxon | Values  | % change | reason | 
-- | -- | -- | -- | --
NCBITaxon:9606\|Homo sapiens | 62447 / 769603 | 8.11% |  |
NCBITaxon:10090\|Mus musculus | 29005 / 552051 | 5.25% | |   
NCBITaxon:9913\|Bos taurus | 22066 / 208681 | 10.57% |   |
NCBITaxon:242507\|Pyricularia oryzae 70-15 | 14170 / 14170 | 100.00% | PANGO ND!?!? | 
NCBITaxon:3218\|Physcomitrium patens | 33171 / 33171 | 100.00% | New IBAs |
NCBITaxon:243230\|Deinococcus radiodurans R1 = ATCC 13939 = DSM 20539 | 2811 / 2811 | 100.00% |  New IBAs|

**Comment by pgaudet** [comment]
* NEW REFERENCES: 2120 - to isoforms
* NEW PMID:1557 
* * PMID: 38382089 >> TAIR, not in QuickGO 
* REMOVED REFERENCES: 15,000 (14.740 MGI)

**Comment by pgaudet** [comment]
New isoform data displayed correctly on AmiGO staging: 


![Image](https://github.com/user-attachments/assets/5676e9c2-e152-4226-827d-ca9222fac5a0)

This data was missing from AmiGO - great that we now have it!

**Comment by pgaudet** [comment]
OK everything looks fine!

**Comment by pgaudet** [comment]
Emailed Sandra to inquire about IntAct & ComplexPortal releases

> 
![image](https://github.com/user-attachments/assets/389b7a4b-c559-443f-9065-b3a1cd116a12)

**Comment by pgaudet** [comment]
@suzialeksander 
Major changes: (you can decide which points are worth mentioning):
- GOC now loads isoform data for human, dog, chicken, pig and cow. This results in about 45,000 new annotations for human only
- A bug has been fixed with respect to how some non-coding RNA are imported, resultsing in about 25,000 new annotations for lncRNA and pre-miRNAs (that used to be converted to the more generic 'gene product') 
- MGI pipeline now ran at GO Central - we now have annotations from Ensembl, and InterPro, EC and RHEA IEAs are now displayed using these sources as 'contributors'
- MGI internal references have been removed from GO products. 
-  ComplexPortal, IntAct and Human Protein Atlas (HPA) has made a new release, increasing the number of protein-protein interactions (in particular for human, +60678, and for mouse, + 3595), and increasing by 14000 the number of human CC.

**Comment by pgaudet** [comment]
The stats looked OK, but we decided not to do the release in the hope that the GPAD/GPI 2.0 would be released soon (although this hasn't yet happenned on 2024-09-24)
